### PR TITLE
Load default flags from shared preferences instead from asset file

### PR DESCRIPTION
**Pull request by nbenbourahla-zz**

### CODE REVIEW

File: flaggr/src/main/java/com/comuto/flag/Flaggr.java [hunk 1]
```diff
@@ -69,6 +69,11 @@ private Flaggr(Context context) {
         flaggrComponent.inject(this);
     }
 
+    public void init(String configUrl, String defaultFlagsFileName, @Nullable final FlagsResultCallback callback) {
+        loadLocalFlags(defaultFlagsFileName);
+        loadConfig(configUrl, defaultFlagsFileName, callback);
```
Comment (RomanWuattier) [inline flaggr/src/main/java/com/comuto/flag/Flaggr.java:74]: `loadConfig` could be private

**Review by RomanWuattier — COMMENTED** [review]
A little suggestion related to functions visibility and LGTM

File: flaggr/src/main/java/com/comuto/flag/Flaggr.java (hunk 1 above)
Comment (nbenbourahla-zz) [reply flaggr/src/main/java/com/comuto/flag/Flaggr.java:74]: Done

**Review by RomanWuattier — APPROVED** [review]

File: flaggr/src/main/java/com/comuto/flag/Flaggr.java [hunk 2]
```diff
@@ -115,6 +118,7 @@ public void onError(Exception ex) {
                 }
             });
         } catch (Exception ex) {
+            loadLocalFlags(defaultFlagsFileName);
```
Comment (ccadoret) [inline flaggr/src/main/java/com/comuto/flag/Flaggr.java:121]: could be removed

**Review by alizeec — APPROVED** [review]

**Review by ccadoret — CHANGES_REQUESTED** [review]

**Pull request merged**